packages/auth/amplify_auth_cognito_dart/lib/src/model/cognito_device_secrets.g.dart

```diff
@@ -76,9 +76,9 @@ class _$CognitoDeviceSecrets extends CognitoDeviceSecrets {
       {required this.deviceGroupKey, required this.deviceKey})
       : super._() {
     BuiltValueNullFieldError.checkNotNull(
-        deviceGroupKey, 'CognitoDeviceSecrets', 'deviceGroupKey');
+        deviceGroupKey, r'CognitoDeviceSecrets', 'deviceGroupKey');
```

Comment:
sorry - what is `r`?

Reply:
Raw string - so it doesn't interpolate or escape characters
https://dart-lang.github.io/linter/lints/use_raw_strings.html